string session_id
string peer_id
string type
string sdp
---
